packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -519,6 +520,7 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
     // Unblock websocket endpoints
     this.socketService.resolveReadyness()
 
+    await this.localDbService.deleteInviteData()
```

Comment:
I think we should leave inviteData always

Reply:
If we want to keep the `inviteData`,
```
  version?: InvitationDataVersion.v2
  cid: string
  token: string
  serverAddress: string
  inviterAddress: string
```
for some reason because we think the original invite data is important, then we can have `Community.inviteData` and `Community.storageService` with the following schema,
```
  token: string
  serverAddress: string
```

Otherwise, if we don't care about saving the original invite data and de-duplication is our focus, we can have something like `Community.inviterAddress` (inviter's Tor address), `Community.inviteCid` and then `Community.storageService` with the schema above.

I think it's probably easiest to fit with the existing PR, to have `Community.inviteData` and `Community.storageService` and accept that there could be some duplication (`token` and `serverAddress` would exist in both, but the `storageService` field is meant to change, while the `inviteData` represents the original invite data which doesn't change).

@ikoenigsknecht do you have any thoughts?

Reply:
The schema for `InviteData` is here: https://github.com/TryQuiet/quiet/blob/68313c24a4d3942ebc054a450b01e081ce659d68/packages/types/src/network.ts#L31